docs/examples/notebooks/audio_decoder.ipynb

```diff
@@ -0,0 +1,243 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "30b7205e-8719-4b16-8839-7b52c435d0da",
+   "metadata": {},
+   "source": [
+    "This example demonstrates how to set up a simple audio decoder pipeline. We load and decode audio data using rocAL. The input data used for this example is a sample speech dataset available in as .wav file)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4225a5b0-85ba-4d65-85b0-a276f82c177d",
+   "metadata": {},
+   "source": [
+    "The following python packages are required to run this example."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "df054e1f-9b06-44f4-9aec-80c8798e3225",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install opencv-python\n",
```

Comment:
how about export ROCAL_DATA_PATH which is needed for running this sample

Reply:
addressed